addons/zeus/functions/fnc_moduleGroupSide.sqf

```diff
@@ -48,7 +48,7 @@ if (GETVAR(_unit,ACE_isUnconscious,false) && {GETMVAR(EGVAR(medical,moveUnitsFro
     {
         private _team = assignedTeam _x;
         [_x] joinSilent _newGroup;
-        _x assignTeam _team;
+        if (_team != "") then { _x assignTeam _team; };
```

Comment:
Shouldn't  "" be treated as "MAIN" or whatever the name for "white" is? With this, "" means "keep the previous", which I don't think is right.

Reply:
it's after joining a new group, so fireteam was reset
doing nothing will leave in main

Reply:
ok

Reply:
I feel like there could be a CBA function for changing group side to properly handle this stuff in a common location.